Foundation/ExtraStringAPIs.swift

```diff
@@ -13,19 +13,9 @@
 // Random access for String.UTF16View, only when Foundation is
 // imported.  Making this API dependent on Foundation decouples the
 // Swift core from a UTF16 representation.
-extension String.UTF16View.Index : Strideable {
```

Comment:
It was here: https://github.com/apple/swift/commit/915923999592c4988fb1e6ec1673dfd46043f316